File: Nível 1/1985 - MacPRONALTS/main.py
numero = int(input("Clc aq: "))

compra = 0

for i in range(1, numero + 1):
    valores = input("A: ")
    x1,x2 = valores.split(" ")
    x1 = int(x1)
    x2 = int(x2)

    if x1 == 1001:
        calculo = 1.50 * x2
        compra += calculo
    elif x1 == 1002:
        calculo = 2.50 * x2
        compra += calculo
    elif x1 == 1003:
        calculo = 3.50 * x2
        compra += calculo
    elif x1 == 1004:
        calculo = 4.50 * x2
        compra += calculo
    elif x1 == 1005:
        calculo = 5.50 * x2
        compra += calculo

print("%.2f" % compra)

File: Nível 1/1985 - MacPRONALTS/main.lua
numero = tonumber(io.read())
compra = 0

for i = 1, numero do 
    valores = io.read()
    x1, x2 = valores:match("(%S+)%s+(%S+)")
    x1,x2 = tonumber(x1), tonumber(x2)

    if x1 == 1001 then
        calculo = 1.50 * x2
        compra = compra + calculo
    elseif x1 == 1002 then
        calculo = 2.50 * x2
        compra = compra + calculo
    elseif x1 == 1003 then
        calculo = 3.50 * x2
        compra = compra + calculo
    elseif x1 == 1004 then
        calculo = 4.50 * x2
        compra = compra + calculo
    elseif x1 == 1005 then
        calculo = 5.50 * x2
        compra = compra + calculo
    end
end

print(string.format("%.2f", compra))